### PR TITLE
Fixed critical error on preflight cal

### PR DESCRIPTION
**Pull request by tridge**
prevent a critical error for ahrstrim

### CODE REVIEW

**Comment by tridge** [comment]
fixes critical error from #12800

File: libraries/GCS_MAVLink/GCS_Common.cpp [hunk 1]
```diff
@@ -3358,6 +3358,7 @@ MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration_baro()
 
 MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration(const mavlink_command_long_t &packet)
 {
+    EXPECT_DELAY_MS(30000);
```
Comment (andyp1per) [inline libraries/GCS_MAVLink/GCS_Common.cpp]: Wow! Why does this only occur on some boards?

Comment (WickedShell) [reply libraries/GCS_MAVLink/GCS_Common.cpp]: Why put this here? Shouldn't the actual calibration methods be annotated appropriately?

Comment (tridge) [reply libraries/GCS_MAVLink/GCS_Common.cpp]: @WickedShell this covers all of them with one call, and the macro allows for both nesting and automatically unsets the timeout when the context is left. This simple approach ensures we don't miss one

**Comment by rmackay9** [comment]
this has been added to Copter-4.0.0-rc3

**Pull request merged**